SECURITY.md

```diff
@@ -0,0 +1,20 @@
+# Eiffel-community Vulnerability and Security Reporting and Response
+
+## Report a Vulnerability or Security Issue
+We as a community encourage researchers, users and contributors to report vulnerabilities and security related issues to the Eiffel community. All issues are thoroughly investigated by a community security officer and/or other community security volunteers. All reported and fixed security and vulnerability issues can be found on the [Eiffel community security page](https://eiffel-community.github.io/security.html) .
+
+## How to Report a Security Vulnerability
+To file a vulnerability report please send and e-mail to the private eiffel-community-security@googlegroups.com  list. The e-mail should list the security specific details as well as the [standard bug report information](https://github.com/eiffel-community/.github/blob/master/.github/ISSUE_TEMPLATE.md). Only the community security officers will have access to e-mails sent on the security and vulnerability list. This process is the same whether the report stems form an project within the Eiffel community or an external contributor. 
+
+Triage and handling of the vulnerability report will be conducted within one week. If the vulnerability severity and impact is high a patch will be published with urgency.
```

Comment:
Before merging this PR I assume we need to appoint the 'security officer' and maybe also a group of 'security volunteers' within the community, to be able to live up to this promise.

Reply:
I think this comment is addressed during the last TC meeting.

I have an action item from the meeting to update the governance document and write the term of security contacts.
I can add Fredrik and Kristofer in that section as well instead of adding to this doc and include link to that section in this doc so the process remains same but the names are handled in governance document?

Reply:
Yes, of course you could link to the governance docs for some of the details in the process. I'm not sure we should/need to disclose any names of the security officers/volunteers in a public document though. Should we? If not, where should we keep such information? What is the best practice in this area? Do most projects disclose their security officers/volunteers publicly?
And for the security mail list I assume this PR depends on #60 ?

Reply:
I've seen both - some communities disclose the names of the team members while others just call "Security Team" without listing people. Putting communities that list individuals as examples.

https://security.openstack.org
https://wiki.jenkins.io/display/JENKINS/Team+Leads

Reply:
Ok. The Jenkins info there is very old, and I think that OpenStack plays in a different league than Eiffel does. I'd vote for that we don't need to keep a public list of security representatives within the community, but for sure we need to have them noted in some internal TC document. But where in that case? HackMD? Google Docs?

Reply:
So, should I link something from the process document or not?

Reply:
I'd say that the governance document that Fatih is about to propose an update to should include some description of what the 'security officer' and 'security volunteer' roles are, and then you should be able to link to those from this document.
And of course the people behind those roles need to be appointed and added to relevant mail lists (?) before this PR is merged, so we can live up to the promises made here. I suggest that we appoint those people on the next TC meeting.

Reply:
We appointed "security officers" - Fredrik and Kristofer. I assume Kristofer made Fredrik a member of the maillist/group he created as well so this PR does not seem to have anything left to be fixed.

As part of the AP I got, I'll update the governance document and add the role "Security Officers" with the responsibilities and terms which is tied to TC term. For that, I need this doc to be merged. To close the gap, I'll add a link to that section from governance document to this document as well which I'll do that within the PR.

I think we should merge this PR and continue the conversation on listing the individuals volunteered/appointed as security officers under the new PR.

Reply:
Yes, we both me and Fredrik are on the list. Ok to merge.

Reply:
Ok. Sounds fine.